### PR TITLE
Replace bs problem comment with a link

### PR DESCRIPTION
**Pull request by manolo**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-core-elements/47)

### CODE REVIEW

File: docs/angular2.adoc [hunk 1]
```diff
@@ -64,16 +64,7 @@ window.addEventListener('WebComponentsReady', function() {
...
-
-[source,javascript]
-----
-{
-  "snippetOptions": {
-    "ignorePaths": "bower_components/**/*.html"
-  }
-}
-----
+*Note*: if you experiment some problems when running your app in devmode, please visit this [known issue](https://github.com/vaadin/vaadin-core-elements/issues/46)
```
Comment (jouni) [inline docs/angular2.adoc]: You could also use the [NOTE] notation.

**Comment by tehapo** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/angular2.adoc, line 67 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-core-elements/47#-KGXLTvkmx8iudY_iNEo-r1-67:-KGkVMm8Iq7q_EUf1TE_:1660947777) ([raw file](https://github.com/vaadin/vaadin-core-elements/blob/4213ba0728df250a67083de36f9420bfbc7913bf/docs/angular2.adoc#L67)):*
Yes, the `[NOTE]` notation would be better here also to be consistent with the other note about Angular 2 version requirement.

Should we also be more explicit about the error? Maybe state that "if you see errors about missing behaviors in the console, please see this known issue" or something like that.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-core-elements/47)*

**Comment by manolo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/angular2.adoc, line 67 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-core-elements/47#-KGXLTvkmx8iudY_iNEo-r1-67:-KGpHjqdGCWp8bjHNYVS:-499146009) ([raw file](https://github.com/vaadin/vaadin-core-elements/blob/4213ba0728df250a67083de36f9420bfbc7913bf/docs/angular2.adoc#L67)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-core-elements/47)*

**Comment by tehapo** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-core-elements/47#-:-KGpJmr3uNYPuebFTiDm:1417088721)*

**Comment by tehapo** [comment]
Oh, noticed that the link syntax was MarkDown instead of AsciiDoc. Fixed that already in this branch.

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-core-elements/47#-:-KGpKRy-YGGKirhiZdyj:935482163)*

**Pull request merged**